fc/fullcontact.go

```diff
@@ -8,6 +8,30 @@ import (
 	"time"
 )
 
+func (fcClient *fullContactClient) newHttpRequest1(url string, reqBytes []byte) (*http.Request, error) {
```

Comment:
If you have combined both request types here, then we can remove individual methods below?

Reply:
I created this method, but have't used it. Forgot about adding this as well :)